loader/loader.c

```diff
@@ -2471,7 +2471,7 @@ static VkResult loader_read_layer_json(const struct loader_instance *inst, struc
             cJSON *inst_version_json = cJSON_GetObjectItem(pre_instance, "vkEnumerateInstanceVersion");
             if (inst_version_json) {
                 char *inst_version_name = cJSON_Print(inst, inst_version_json);
-                if (inst_version_json) {
+                if (inst_version_json == NULL) {
```

Comment:
Now that I think about it, shouldn't this be 
`if (inst_version_name == NULL) {`?